MonoGame.Framework/Graphics/GraphicsAdapter.DirectX.cs

```diff
@@ -115,12 +115,25 @@ private bool PlatformIsProfileSupported(GraphicsProfile graphicsProfile)
         {
             if(UseReferenceDevice)
                 return true;
+
+            FeatureLevel highestSupportedLevel;
+            try
+            {
+                highestSupportedLevel = SharpDX.Direct3D11.Device.GetSupportedFeatureLevel();
+            }
+            catch (SharpDX.SharpDXException ex)
+            {
+                if (ex.ResultCode == SharpDX.DXGI.ResultCode.Unsupported) // No supported feature levels!
+                    return false;
+                throw;
+            }
+
             switch(graphicsProfile)
             {
                 case GraphicsProfile.Reach:
-                    return SharpDX.Direct3D11.Device.IsSupportedFeatureLevel(_adapter, FeatureLevel.Level_9_1);
+                    return (highestSupportedLevel >= FeatureLevel.Level_9_1);
                 case GraphicsProfile.HiDef:
-                    return SharpDX.Direct3D11.Device.IsSupportedFeatureLevel(_adapter, FeatureLevel.Level_10_0);
+                    return (highestSupportedLevel >= FeatureLevel.Level_10_0);
```

Comment:
@nkast I did not realise this! But I'd definitely rather leave the 9.3 compatibility stuff for a separate PR. Otherwise I'm worried it may delay this fix being merged.

Reply:
No worries.